### PR TITLE
Added font-sizes variables

### PR DESCRIPTION
**Pull request by tureey**
Added Font sizes variables specified in *Figma*

```
// Font-sizes
$fz-xxs: 0.75rem; // 12px
$fz-xs: 0.875rem; // 14px
$fz-s: 1rem; // 16px
$fz-m: 1.125rem; // 18px
$fz-l: 1.5rem; // 24px
$fz-xl: 2rem; // 32px
$fz-xxl: 4.25rem; // 68px
$fz-giant: 4.25rem; // 84px
```

### CODE REVIEW

File: src/styles/scss/base/_variables.scss [hunk 1]
```diff
@@ -1,4 +1,17 @@
...
+
+// Font-sizes
+$fz-xxs: 0.75rem; // 12px
+$fz-xs: 0.875rem; // 14px
+$fz-s: 1rem; // 16px
+$fz-m: 1.125rem; // 18px
+$fz-l: 1.5rem; // 24px
+$fz-xl: 2rem; // 32px
+$fz-xxl: 4.25rem; // 68px
+$fz-giant: 4.25rem; // 84px
```
Comment (palashmon) [inline src/styles/scss/base/_variables.scss]:
```scss
$fz-xxl: 4.25rem; // 68px
$fz-giant: 4.25rem; // 84px
```
Looks like both `xxl` & `giant` font-size has same with `4.25rem` here. `$fz-giant` should have relatively higher width, right?

Comment (tureey) [reply src/styles/scss/base/_variables.scss]: Yes, my bad!  it was 5.*rem Good check! @palashmon

**Comment by tureey** [comment]
Sure thing, this evening I'll try t do it

**Comment by emmawedekind** [comment]
Awesome work guys, let's keep it up!

**Comment by tureey** [comment]
@crysfel that's weird because I had no conflicts.. When I brought the development updated to my `branch`.
**Check that everything is okay though**. **If not**, it **will be easier** to delete this `Pull Request`, create a new one with the new folder structure and edit the `variables.scss`

**Comment by crysfel** [comment]
There are several commits from other people in this PR, it seems like you are having the same problems as @nitish24p did in his last PR.

@nitish24p can you please help @tureey on this?

**Comment by tureey** [comment]
Victor ribero in slack if needed 🙂 @nitish24p

**Comment by nitish24p** [comment]
@tureey I did a hard reset to the commit before i did the rebase, then did a force push and then rebased my branch again. :)

**Comment by tureey** [comment]
I was having issues so I just reinstalled the repository.
**Close this PR please**
**new PR:** https://github.com/Coding-Coach/coding-coach/pull/72

**Pull request closed without merging**